clients/benchmarks/client.cpp

```diff
@@ -57,7 +70,7 @@ template <typename Ti, typename To, typename Tc>
 struct perf_gemm_ex<Ti,
                     To,
                     Tc,
-                    typename std::enable_if<!std::is_same<Ti, void>{} && !is_complex<Ti>>::type>
+                    typename std::enable_if<!std::is_same<Ti, void> {} && !is_complex<Ti>>::type>
```

Comment:
The extra spaces should not have been added back.
Are the git hooks using clang 9?

Reply:
The git hooks are using clang 9...not sure what to make of that. Not sure what happened. I've fixed the change in my 2nd commit.